visual_behavior/data_access/loading.py

```diff
@@ -143,9 +143,10 @@ def get_released_ophys_experiment_table(exclude_ai94=True):
     Returns:
         experiment_table -- returns a dataframe with ophys_experiment_id as the index and metadata as columns.
     '''
-    data_storage_directory = '//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/production_cache'
+    # data_storage_directory = '//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/production_cache'
+    # cache = bpc.from_s3_cache(cache_dir=data_storage_directory)
 
-    cache = bpc.from_s3_cache(cache_dir=data_storage_directory)
+    cache = bpc.from_lims(data_release_date='2021-03-25')
```

Comment:
I'm not sure when each loading method should be used. Is "from_s3_cache" outdated?

Reply:
i think the data_storage_directory is what threw me off here, since it is directing the function to a file on the network instead of s3. I will investigate that, it could be that the cache_dir input does nothing when you load from s3.

Reply:
working this out led to somewhat of a rabbit hole that made me reconfigure `get_filtered_ophys_experiment_table()`....i'll as for your review on those revisions in a second. it shouldn't affect what comes out of that function drastically, it was more about the logic of all the if statements in that method.